frontend/src/bundles/common/pages/not-found/not-found.tsx

```diff
@@ -0,0 +1,45 @@
+import { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import {
+    Button,
+    ButtonSize,
+    ButtonVariant,
+} from '../../components/components.js';
```

Comment:
It is better to use relative paths like `'~/bundles/common/components/components.js'`